server/services/caService.js

```diff
@@ -41,12 +43,17 @@ module.exports = function createCaService(roService, lduActiveClient, { continue
       const { lduCode, isAllocated } = ro
 
       const isLduActive = await lduActiveClient.isLduPresent(lduCode)
+
+      if (!isLduActive && !isAllocated) {
+        return { LDU_INACTIVE, COM_NOT_ALLOCATED }
+      }
+
       if (!isLduActive) {
-        return LDU_INACTIVE
+        return { LDU_INACTIVE }
       }
 
       if (!isAllocated) {
-        return COM_NOT_ALLOCATED
+        return { COM_NOT_ALLOCATED }
       }
```

Comment:
could potentially merge these 3 `if`s into:
```js
  return [
     ...(isLduActive ? [LDU_INACTIVE] : []),
     ...(isAllocated ? [COM_NOT_ALLOCATED] : [])
   ]
```

Reply:
worked like this

 if (!isLduActive || !isAllocated) {
 return [...(!isLduActive ? [LDU_INACTIVE] : []), ...(!isAllocated ? [COM_NOT_ALLOCATED] : []) 
  ] }

Reply:
should be able to tidy this up by return empty array rather than null when we want to model "no messages" as `empty array` rather than `null`

Reply:
done